src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs

```diff
@@ -220,7 +228,7 @@ private IServiceProvider ConfigureIOC(IServiceCollection services)
                 option.UseSqlServer(sqlConnectionString, options => options.EnableRetryOnFailure(3));
 
                 config.For<AssessorDbContext>().Use(c => new AssessorDbContext(option.Options));
-                config.For<IDbConnection>().Use(c => new SqlConnection(sqlConnectionString));
+                config.For<IDbConnection>().Use(c => ManagedIdentitySqlConnection.GetSqlConnection(sqlConnectionString, _azureServiceTokenProvider));
```

Comment:
Following on from Adam's [comment](https://github.com/SkillsFundingAgency/das-assessor-service/pull/1057/files#r496557271), we need to understand and test what uses this db connection service.  If it's something like NServiceBus issues with token expiry might not be surfaced just by waiting 1 hour

Reply:
It looks like ManagedIdentitySqlConnection.GetSqlConnection is called per request, so GetAccessTokenAsync would be called at the same time.